main.py

```diff
@@ -764,22 +764,5 @@ def ui_access_wrapper(self, *args, **kwargs):
 # 404 error handler (Needs to be at the end of the URLS list).
 URLS.append(get_redirect_route(r'/<:.*>', base.Error404Handler))
 
-URLS_TO_SERVE = []
-
-if (feconf.ENABLE_MAINTENANCE_MODE and
-        not current_user_services.is_current_user_super_admin()):
-    # Show only the maintenance mode page.
-    URLS_TO_SERVE = [
-        get_redirect_route(r'%s' % feconf.ADMIN_URL, admin.AdminPage),
-        get_redirect_route(r'/adminhandler', admin.AdminHandler),
-        get_redirect_route(r'/adminrolehandler', admin.AdminRoleHandler),
-        get_redirect_route(r'/adminjoboutput', admin.AdminJobOutputHandler),
```

Comment:
Are w34e changing the behavior here? i.e, whether this page was accessible to the admin users (or any other authorized user expert super_admins) earlier?

Reply:
Hmmm, a bit, but I think that it is fine to allow only the super admins (releasers) to the page and deny everyone else.